typings.json

```diff
@@ -6,10 +6,13 @@
   },
   "globalDependencies": {
     "body-parser": "registry:dt/body-parser#0.0.0+20160619023215",
+    "cookies": "registry:dt/cookies#0.5.1+20160316171810",
```

Comment:
I guess this is needed by koa typings. If I remove it, I get this error
```
import * as cookies from "cookies";

typings/globals/koa/index.d.ts(5,30): error TS2307: Cannot find module 'cookies'.
```
I am a typescript beginner so I might be doing something wrong. @nnance can you please check this?